ang/civicase/case/actions/services/email-managers-case-action.service.js

```diff
@@ -8,7 +8,8 @@
    */
   function EmailManagersCaseAction () {
     /**
-     * Click event handler for the Action
+     * Click event handler for the Action.
+     * Spits error if no case manager, opens popup otherwise
```

Comment:
Can we use "displays" instead of "spits" ?

Reply:
I think a better description would be:

```
Returns the configuration options to open up a mail popup to communicate with the case managers. Displays an error message when no case managers have been assigned to the case.
```
I think it's important to define what the function actually does first, and then any side effects second.